superset/db_engine_specs/base.py

```diff
@@ -1283,9 +1284,10 @@ def df_to_sql(
             catalog=table.catalog,
             schema=table.schema,
         ) as engine:
-            if engine.dialect.supports_multivalues_insert:
+            if getattr(engine.dialect, "supports_multivalues_insert", False) or getattr(
+                cls, "supports_multivalues_insert", False
+            ):
```

Comment:
I think you can simplify more (as we've added `supports_multivalues_insert` to `BaseEngineSpec`, it'll always be defined, so no need to `getattr`):
```suggestion
            if getattr(engine.dialect, "supports_multivalues_insert", False) or cls.supports_multivalues_insert:
```
Also, something worth checking, do we really need to `getattr(engine.dialect, "supports_multivalues_insert", False)`, or would `engine.dialect.supports_multivalues_insert` be sufficient (I kinda assume it'll always be defined)? I can help checking this if needed.

Reply:
Agreed that we can go back to `engine.dialect.supports_multivalues_insert` -- that's what it was before this PR so I feel good simplifying it to that.